### PR TITLE
updated homework

### PR DESCRIPTION
**Pull request by moniqueking57**
Amended.

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,77 @@
...
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+Is where you copy the repo to your local machine. 
 
 ## Question 11
 
 What is a Pull Request?
```
Comment (ATedy) [inline homework.md:69]: Hey Monique, nice work. I see you left the personal details and home work details empty at the start. You might have done it intentionally or haven't notice, adding that would be great as they ask in the assignment.

Comment (moniqueking57) [reply homework.md:69]: Hi Amanuel, I did miss those. Thank you for noticing, I have corrected :)

File: homework.md [hunk 2]
```diff
@@ -104,10 +104,10 @@ Microsoft Word
...
 
-To record what changes you made to the code
+It saves your changes to the local repository. 
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Git Cheatsheet! Or the CYF slack channel :)
```
Comment (nirmeet-baweja) [inline homework.md:113]: I liked the fact that you added the CYF slack channel in your answer!

File: homework.md [hunk 3]
```diff
@@ -4,77 +4,77 @@
...
+    whatever command you would like. 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+It is a text-based programming language.
```
Comment (nirmeet-baweja) [inline homework.md:45]: Hi Monique! Well done! You could include a bit more detail about JavaScript here. Like the fact that it is generally used for web-pages or some other similar fact.

**Comment by nirmeet-baweja** [comment]
I have finished my review.

**Comment by moniqueking57** [comment]
Thank you :)

**Pull request closed without merging**